blockchain/mining-token-rewards.md

```diff
@@ -26,7 +26,7 @@ Every [epoch](mining-token-rewards.md), the current consensus group mines some a
 | Network Data Transfer | Distributed each epoch to Hotspots that route LongFi sensor data for sensors on the Network during that epoch.  |
 
 {% hint style="info" %}
-**Do I Have To Actively Participate to Earn Rewards Once My Hotspot is Deployed?**
+**Do I Have To Actively Participate to Earn Rewards Once My Hotspot is deployed?**
```

Comment:
It's odd that this is all title case except for the last word.

Reply:
Oops. Good catch. I was sort of removing title case from things but then decided to revert it to keep things consistent with the rest of the docs. Missed one word ;)